### PR TITLE
Add missing copyright headers

### PR DESCRIPTION
**Pull request by isle2983**
Adds 'The Bitcoin Core developers' copyright headers in 12 source files.
<start_year>-<end_year> is as per the history obtained from 'git log'.

The users to have touched them in the past are listed below.  This was last
discussed in pull request  #7300:

In that discussion I can see perma-ACKs from @MarcoFalke, @gavinandresen and
@laanwj.

So that leaves ACKs needed from:
@sipa
@jgarzik
@luke-jr
@theuni
@ChoHag 
@calebogden
@mrbandrews
@casey
@mitchellcash
@brandondahler
Phillip Kaufmann

Broken down by file:

contrib/devtools/optimize-pngs.py (@MarcoFalke, @jonasschnelli, @laanwj)
contrib/devtools/security-check.py (@ChoHag, @laanwj, @theuni, @calebogden)
contrib/devtools/test-security-check.py (@ChoHag, @laanwj)
contrib/qt_translations.py (@gavinandresen)
contrib/seeds/makeseeds.py (@laanwj, @sipa, @gavinandresen)
contrib/spendfrom/setup.py (@gavinandresen)
contrib/spendfrom/spendfrom.py (@gavinandresen)
contrib/testgen/base58.py (@laanwj)
contrib/testgen/gen_base58_test_vectors.py (@laanwj)
contrib/zmq/zmq_sub.py (@jonasschnelli, @jgarzik)
qa/rpc-tests/test_framework/blockstore.py (@mrbandrews, @MarcoFalke, @casey, @jonasschnelli)
share/qt/extract_strings_qt.py (@ChoHag, @mitchellcash, @laanwj, @luke-jr, @brandondahler, Phillip Kaufmann, @theuni)

### CODE REVIEW

**Comment by casey** [comment]
Ack from me

**Comment by brandondahler** [comment]
permanent ACK on all code I contribute to this repository having the preferred copyright of the project

**Comment by sipa** [comment]
ack from me

**Comment by jonasschnelli** [comment]
ACK

**Comment by fanquake** [comment]
ACK 2229ffa

**Comment by mrbandrews** [comment]
ack

**Comment by theuni** [comment]
ACK 2229ffa5e2b8e38cd962c02e2fc892e0c925675b

**Comment by MarcoFalke** [comment]
utACK 2229ffa5e2b8e38cd962c02e2fc892e0c925675b

Could make sense to explicitly add a section to https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#contributing-to-bitcoin-core that says that every contribution must be MIT licensed by its author prior to creating a pull request.

**Comment by binflyingup** [comment]
ACK 2229ffa

**Comment by calebogden** [comment]
ACK 2229ffa5e2b8e38cd962c02e2fc892e0c925675b

**Comment by isle2983** [comment]
Thanks everyone!

We have ACK quorum on:

contrib/devtools/optimize-pngs.py
contrib/qt_translations.py
contrib/seeds/makeseeds.py
contrib/spendfrom/setup.py
contrib/spendfrom/spendfrom.py
contrib/testgen/base58.py
contrib/testgen/gen_base58_test_vectors.py
qa/rpc-tests/test_framework/blockstore.py

Still waiting on:

contrib/zmq/zmq_sub.py (@jgarzik)
contrib/devtools/test-security-check.py (@ChoHag)
contrib/devtools/security-check.py (@ChoHag)
share/qt/extract_strings_qt.py (@ChoHag, @mitchellcash, @luke-jr, Phillip Kaufmann)

I will give it a couple more days and then split the remaining files into a different PR so the ACKed ones can go in.

**Comment by achow101** [comment]
Add the copyright to https://github.com/bitcoin/bitcoin/blob/master/contrib/gitian-build.sh please. I'll ACK it.

**Comment by isle2983** [comment]
@achow101 copyright headers gitian-buiild.sh added in dffe727b259d0cc564e86aa1c33e7285e0f8c5b0

...which reminds me that I should do another PR for *.sh files.

**Comment by mitchellcash** [comment]
ACK https://github.com/bitcoin/bitcoin/commit/dffe727b259d0cc564e86aa1c33e7285e0f8c5b0

**Comment by luke-jr** [comment]
ACK

**Comment by isle2983** [comment]
783e930e68a312bc7654d833053cceead18dd688 drops the addition of the MIT headers in these files due to missing ACKs listed here:

contrib/zmq/zmq_sub.py (@jgarzik)
contrib/devtools/test-security-check.py (@ChoHag)
contrib/devtools/security-check.py (@ChoHag)
share/qt/extract_strings_qt.py (@ChoHag, Phillip Kaufmann)

I will re-submit those as different PRs.


@achow101 still owes the ACK promised above for gitian-build.sh. Assuming that comes, I submit that this PR is ready given the ACKs in  #7300 and the present thread above.

To recap, those were:

contrib/devtools/optimize-pngs.py (@MarcoFalke, @jonasschnelli, @laanwj)
contrib/gitian-build.sh (@achow101)
contrib/qt_translations.py (@gavinandresen)
contrib/seeds/makeseeds.py (@laanwj, @sipa, @gavinandresen)
contrib/spendfrom/setup.py (@gavinandresen)
contrib/spendfrom/spendfrom.py (@gavinandresen)
contrib/testgen/base58.py (@laanwj)
contrib/testgen/gen_base58_test_vectors.py (@laanwj)
qa/rpc-tests/test_framework/blockstore.py (@MarcoFalke, @jonasschnelli, @casey, @mrbandrews)

**Comment by achow101** [comment]
ACK (I thought my ACK was implied by my statement)

**Comment by MarcoFalke** [comment]
I think this is ready

ACK dffe727
